contributing/submitting-pr.md

```diff
@@ -202,7 +247,49 @@ Then ensure the menu displays the original repository (`ploomber/PROJECT_NAME`)
 base repository: ploomber/PROJECT_NAME base: main (or master) ⬅️ head repository: YOURUSERNAME/PROJECT_NAME compare: YOURBRANCH
 ```
 
-Then click on open Pull Request.
+Then click on open **Draft Pull Request.** Opening a PR will trigger building
+documentation and running the tests. Wait for a few minutes (most of our tests run in
+<15 minutes). If the checks failed:
+
```

Comment:
Screenshot of `convert to draft` link might also help (in case user opens a regular PR)  by mistake

Reply:
added